tsconfig.json

```diff
@@ -12,6 +12,7 @@
     "noEmit": true,
     "skipLibCheck": true,
     "resolveJsonModule": true,
+    "strict": true,
```

Comment:
Why is tsconfig modified here?

Reply:
Good catch! Tror den er igjen etter jeg lekte meg litt rundt med forskjellige pakker. Fjerner 😊